src/main/java/com/google/sps/GetConfigProperties.java

```diff
@@ -0,0 +1,38 @@
+package com.google.sps;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Properties;
+
+public class GetConfigProperties {
```

Comment:
Could you also add some javadoc style comments to the functions here explaining what they're doing and what the inputs/outputs are.